features/step_definitions/catalogue_steps.rb

```diff
@@ -47,18 +47,20 @@
 end
 
 Then (/^I note the number of search results$/) do
-  @service_count = CatalogueHelpers.get_service_count(page)
+  @result_count = CatalogueHelpers.get_service_count(page)
+  puts "Noted result_count: #{@result_count}"
 end
 
 Then /^I click a random category link$/ do
   links = CatalogueHelpers.get_category_links(page)
   link_el = links.sample
   @category_name = link_el.text.sub(/ \([0-9]*\)/, "")  # remove service count
+  puts "Clicking '#{link_el.text}'"
   link_el.click
 end
 
 Then(/^I see fewer search results than noted$/) do
-  expect(CatalogueHelpers.get_service_count(page)).to be < @service_count
+  expect(CatalogueHelpers.get_service_count(page)).to be < @result_count
```

Comment:
`get_result_count` ?

Reply:
Yeah I could have dug further to unify the two sets of steps, but I was really just interested in getting the variable name the same so that they could share *some* steps.